stellargraph/layer/gcn_lstm.py

```diff
@@ -332,16 +281,69 @@ def __init__(
         self.bias_constraint = constraints.get(bias_constraint)
 
-        self.gcn_1 = GraphConvolution(units = layer_sizes[0], A=adj, activation=self.activations[0])
-        self.gcn_2 = GraphConvolution(units = layer_sizes[0], A=adj, activation=self.activations[1])
-        self.lstm = LSTM(self.layer_sizes[1], return_sequences=False)
-        self.dense = Dense(self.outputs, activation=self.activations[2])
+        # Activation function for each gcn layer
+        if gc_activations is None:
+            gc_activations = ["relu"] * n_gc_layers
+        elif len(gc_activations) != n_gc_layers:
+            raise ValueError(
+                "Invalid number of activations; require one function per graph convolution layer"
+            )
+        self.gc_activations = gc_activations
+
+        # Activation function for each gcn layer
+        if lstm_activations is None:
+            lstm_activations = ["tanh"] * n_lstm_layers
+        elif len(lstm_activations) != n_lstm_layers:
+            raise ValueError(
+                "Invalid number of activations; require one function per lstm layer"
+            )
+        self.lstm_activations = lstm_activations
+
+        self.gcn_1 = FixedAdjacencyGraphConvolution(
+            units=self.gc_layer_sizes[0], A=self.adj, activation=self.gc_activations[0]
+        )
+        self.gcn_2 = FixedAdjacencyGraphConvolution(
+            units=self.gc_layer_sizes[1], A=self.adj, activation=self.gc_activations[1]
+        )
+        self.lstm = LSTM(
+            self.lstm_layer_sizes[0],
+            activation=self.lstm_activations[0],
+            return_sequences=False,
+        )
+        self.dense = Dense(self.outputs, activation="sigmoid")
         self.dropout = Dropout(self.dropout)
 
-    def call(self, inputs):
-        layer = self.gcn_1(inputs)
-        layer = self.gcn_2(layer)
-        layer = self.lstm(layer)
-        layer = self.dropout(layer)
-        layer = self.dense(layer)
-        return layer
+    def __call__(self, x):
+
+        x_in, out_indices = x
+
+        h_layer = x_in
+        h_layer = self.gcn_1(h_layer)
+        h_layer = self.gcn_2(h_layer)
+        h_layer = self.lstm(h_layer)
+        h_layer = self.dropout(h_layer)
+        h_layer = self.dense(h_layer)
+
+        return h_layer
+
+    def in_out_tensors(self):
+        """
+        Builds a GCN model for node  feature prediction
+
+        Returns:
+            tuple: `(x_inp, x_out)`, where `x_inp` is a list of Keras/TensorFlow
+            input tensors for the GCN model and `x_out` is a tensor of the GCN model output.
+        """
+        # Inputs for features
+        x_t = Input(batch_shape=(None, self.n_features, self.n_nodes))
+
+        # Indices to gather for model output
+        out_indices_t = Input(batch_shape=(None, self.n_nodes), dtype="int32")
+
+        x_inp = [x_t, out_indices_t]  # + A_placeholders
+        x_out = self(x_inp)
+
+        return x_inp[0], x_out
+
+    def _node_model(self):
```

Comment:
looks like `_node_model` is an unused hidden function that wraps `in_out_tensors` - I think this can be safely deleted

```suggestion
```